src/main/java/com/sureit/admin/attendance/controller/EmployeeController.java

```diff
@@ -22,86 +22,71 @@
 @RestController("/")
 @EnableAutoConfiguration
 public class EmployeeController {
-	
+
 	@Autowired
 	private EmployeeService employeeService;
 
 	@RequestMapping("/hello")
-	public @ResponseBody String message() {
-		return "simple message";
+	public String getHelloMessage() {
+		return "Hellow World";
 	}
 
-	@RequestMapping("/empList")
-	public @ResponseBody List<Employee> getAllEmployees() {
-		// TODO call business layer service method
+	@CrossOrigin
+	@RequestMapping("/getEmployeeList")
+	public @ResponseBody Map<String, Object> getAllEmployees() {

```

Comment:
Just pass a call to EmployeeService instead doing computation here. What is the purpose of sending count as response ?